checker/check_result.go

```diff
@@ -56,6 +56,48 @@ type DetailLogger interface {
 	Debug(desc string, args ...interface{})
 }
 
+// FileType is the type of a file.
+type FileType int
```

Comment:
Is there a specific reason to create a custom type for int?

Reply:
to have the compiler verify it's an enum